### DISCUSSION TITLE
Graph-API support for Roaming Signatures

### DISCUSSION THREAD

**Discussion by DewinU** [microsoftgraph/msgraph-sdk-dotnet#2446] [Q&A]
Since roaming signatures are a thing now for outlook, it would be nice to also have a way to programmatically change the signatures of the users in AD using the graphAPI, i see that we have MailboxSettings, but there's no option to edit the signature

**Comment by andrueastman** [comment]
Thanks for raising this @DewinU 

As this is an API related question, any chance you could post it at the link below to et feedback from the API owners? As this repo is mainly for SDK related issues, we may not be best placed to answer these questions.

https://learn.microsoft.com/en-us/answers/tags/161/ms-graph
